source/exploring_catalog_and_datasets/03_searching_the_data/search.rst

```diff
@@ -42,6 +42,14 @@ Searching in the data
      * ``#null(referer)``
 
        returns every record which has no value in the column named referer
+   * * containing a value that starts with the specified string, in the specified field
+     * #startswith(id_field,string)
+
+       .. note::
+          ``#startswith`` queries are case sensitive.
+     * ``#startswith(user_id,anon)``
```

Comment:
I works juste fine, but I was wondering if we should not use `#startswith(user_id,"anon")` with the string between double quote, to match the `#exact` syntax and enforce the type.

Reply:
Ok, I'll change the syntax to `#startswith(user_id,"anon")` and remove the note, given that we use the `#exact` syntax.